src/TermsAndConditionsModal.tsx

```diff
@@ -0,0 +1,107 @@
+import { Button, KIND } from "baseui/button";
+import { Modal } from "baseui/modal";
+import { useCallback, useState } from "react";
+import { useTranslation } from "react-i18next";
+
+type Props = {
+  termsAndConditionsIsOpen: boolean;
+  setTermsAndConditionsIsOpen: (isOpen: boolean) => void;
+};
+
+const TermsAndConditionsModal = (props: Props) => {
+  const { setTermsAndConditionsIsOpen } = props;
+
+  const { t } = useTranslation("common");
+
+  const close = useCallback(() => {
+    setTermsAndConditionsIsOpen(false);
+  }, [setTermsAndConditionsIsOpen]);
+
+  return (
+    <Modal
+      onClose={close}
+      isOpen={!!props.termsAndConditionsIsOpen}
+      unstable_ModalBackdropScroll={true}
+      overrides={{
+        Root: { style: { zIndex: 1500 } },
+        Dialog: {
+          style: {
+            height: "auto",
+            width: "600px",
+            display: "flex",
+            flexDirection: "column",
+            padding: "1.5rem",
+          },
+        },
+      }}
+    >
+      <h2>Terms and Conditions</h2>
+      <p
+        style={{
+          marginTop: "1rem",
+          marginBottom: "0.5rem",
+          marginRight: "1rem",
+        }}
+      >
+        Welcome to vaxx.nz These terms and conditions outline the rules and
+        regulations for the use of vaxx.nz. By accessing this website we assume
+        you accept these terms and conditions. Do not continue to use if you do
+        not agree to take all of the terms and conditions stated on this page.
+        The following terminology applies to these Terms and Conditions, Privacy
+        Statement and Disclaimer Notice and all Agreements: “You” refers to you,
+        the person log on this website and compliant to the Company's terms and
```

Comment:
"the person using this website and compliant to the site's terms and conditions."